Library/ViewModels/PledgePaymentPlansCellViewModel.swift

```diff
@@ -0,0 +1,63 @@
+import ReactiveSwift
+
+public typealias PledgePaymentPlanCellData = (
+  type: PledgePaymentPlansType,
+  isSelected: Bool
+)
+
+public protocol PledgePaymentPlansCellViewModelInputs {
+  func configureWith(data: PledgePaymentPlanCellData)
+}
+
+public protocol PledgePaymentPlansCellViewModelOutputs {
+  var titleText: Signal<String?, Never> { get }
+  var subtitleText: Signal<String?, Never> { get }
+  var checkmarkImageName: Signal<String, Never> { get }
+}
+
+public protocol PledgePaymentPlansCellViewModelType {
+  var inputs: PledgePaymentPlansCellViewModelInputs { get }
+  var outputs: PledgePaymentPlansCellViewModelOutputs { get }
+}
+
+public final class PledgePaymentPlansCellViewModel: PledgePaymentPlansCellViewModelType,
+  PledgePaymentPlansCellViewModelInputs,
+  PledgePaymentPlansCellViewModelOutputs {
+  public init() {
+    let checkImageName = self.configData.signal
+      .map { $0?.isSelected ?? false ? "icon-payment-method-selected" : "icon-payment-method-unselected" }
+
+    self.checkmarkImageName = checkImageName
+
+    self.titleText = self.configData.signal.skipNil().map { getTitleText(by: $0.type) }
+    self.subtitleText = self.configData.signal.skipNil().map { getSubtitleText(by: $0.type) }
+  }
+
+  fileprivate let configData = MutableProperty<PledgePaymentPlanCellData?>(nil)
```

Comment:
Great question! The reason this defaults to `nil` is because there’s no default information to display until the option is explicitly configured. Setting it to `nil` ensures that the option remains in an unconfigured state until `configureWith` is called with the appropriate data.

Additionally, using `nil` as a default value is a common pattern I’ve seen used in many `configureWith` functions across the project, as it helps enforce a clear lifecycle: data is set explicitly when available, and we avoid assuming a placeholder or default state that might lead to unintended behavior. Let me know if you think there’s a better approach here, I’m happy to discuss further!